packages/features/ee/workflows/lib/test/workflows.test.ts

```diff
@@ -86,17 +86,17 @@ const mockBookings = [
     eventTypeId: 1,
     userId: 101,
     status: BookingStatus.ACCEPTED,
-    startTime: `2024-05-22T04:00:00.000Z`,
-    endTime: `2024-05-22T04:30:00.000Z`,
+    startTime: `2024-05-20T14:00:00.000Z`,
```

Comment:
make date closer to the current date `2024-05-20T11:59:59Z` so we can test if scheduling works